packages/next/src/build/templates/app-page.ts

```diff
@@ -317,6 +311,11 @@ export async function handler(
     ssgCacheKey = resolvedPathname
   }
 
+  let staticPathKey = ssgCacheKey
```

Comment:
this confuses me a bit since conceptually I'd expect `staticPathKey` and `ssgCacheKey` to be the same - probably worth a comment